product_abc_classification/models/product_product.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 ForgeFlow
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    abc_classification_profile_id = fields.Many2one(
```

Comment:
Hi. 

I don't understand why the abc_classification_profile_id is in the variant level, and not in the template level. Does it make sense to set a profile 1 for a variant, and a profile 2 to another variant of the same template ? 

Propose to move the profile in the template level. (and just set a related in the variant level).

What do you think ? 

Also, it could be great to have an api@depends("categ_id") to set the correct profile automatically when creating a new product, linked to a product_category with a profile defined.

Reply:
Sorry, I didn't have time to check this comment. Well, now if you think the code can be improved, then you can propose a PR for this version or do the refactor in the next one.

Reply:
Arf... this require to write migration script to change the model.

I'll see. 

But well, it doesn't answer to my question : Does it make sense to set a profile 1 for a variant, and a profile 2 to another variant of the same template ?

Thanks.

Reply:
@legalsylvain From my POV it makes more sense in the variant. You can have a variant that is more successful in the market than other, and therefore you classify one as A and other as B.

My 2 cents.

Reply:
Not sure you get it.
Of course the level a/b/c should be in the variant.
But the perfil is common for all the variant of a product template, by design. 

Concrete counter exemple welcome.